flag-app/flag-app-project/src/context/CountriesContext.tsx

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+import { Country } from "../interfaces";
```

Comment:
When you have only 2 imports like here, you don't have to separate them with empty lines